galaxy/api/v2/urls.py

```diff
@@ -36,4 +38,7 @@
     path('collection-versions/<int:pk>/',
          views.CollectionVersionView.as_view(),
          name='collection-version-detail'),
+    path('collection-versions/<int:pk>/artifact/',
+         views.CollectionArtifactView.as_view(),
+         name='collection-version-artifact')
```

Comment:
Probably want the view `name` to be unique in case referring to them, such as with `reverse()`

Reply:
According to django docs it's not necessary: https://docs.djangoproject.com/en/2.2/topics/http/urls/#naming-url-patterns

> You may also use the same name for multiple URL patterns if they differ in their arguments. In addition to the URL name, reverse() matches the number of arguments and the names of the keyword arguments.